source/grid/grid_generator.cc

```diff
@@ -4317,6 +4331,18 @@ namespace GridGenerator
 }
 
 // explicit instantiations
+namespace GridGenerator
+{
+
+  template void
+  hyper_sphere< 1 ,  2 > (Triangulation< 1 ,   2> &,
+                          const Point<2>   &center = Point<2>(),
+                          const double        radius = 1.);
```

Comment:
Can you come up with a follow up patch that removes the default values of the arguments here and in the next couple of lines to silence the earnings @tamiko observes?